src/fixes/quote_character.rs

```diff
@@ -0,0 +1,94 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct QuoteCharacterFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for QuoteCharacterFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "QuoteCharacter",
+        }
+    }
+}
+
+impl Fix for QuoteCharacterFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_line(&self, line: &mut LineEntry) -> Option<()> {
+        let value = line.get_value()?;
+        let pure_val = value.replace("'", "").replace("\"", "");
+
+        line.raw_string = format!("{}={}", line.get_key()?, pure_val);
+
+        Some(())
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::path::PathBuf;
+
+    #[test]
+    fn fix_line_test() {
+        let fixer = QuoteCharacterFixer::default();
+        let mut line = LineEntry {
+            number: 1,
+            file: FileEntry {
+                path: PathBuf::from(".env"),
+                file_name: ".env".to_string(),
+                total_lines: 1,
+            },
+            raw_string: String::from("foo=\'\"b\'\"ar\"\'"),
+        };
+        assert_eq!(Some(()), fixer.fix_line(&mut line));
+        assert_eq!("foo=bar", line.raw_string);
```

Comment:
```suggestion
        assert_eq!("FOO=bar", line.raw_string);
```
The key should be in uppercase.